docs/w3c.md

```diff
@@ -0,0 +1,16 @@
+Perform touch actions for W3C module. Generate `touch` pointer action here and users can use this via `driver.action`
+
+[Documentation](http://www.rubydoc.info/github/appium/ruby_lib_core/Appium%2FCore%2FBase%2FCoreBridgeW3C:action)
```

Comment:
I think it makes sense to explicitly mention that relative action coordinates are counted from the center of element's rectangle unlike in jsonwp standard, where coordinates are counted from the top left point.

The second important restriction: only `touch` pointer type is supported for iOS.